### ISSUE TITLE
Dropout rate

### ISSUE THREAD

**Issue by sh0416** [chenchongthu/DeepCoNN#20]
Can I ask you why you use drop rate with 1.0??

I am new to tensorflow, and I found that it means you make all your feature goes to zero.

I don't understand why you choose this..

```
with tf.name_scope("dropout"):
            self.h_drop_u = tf.nn.dropout(self.h_pool_flat_u, 1.0)
            self.h_drop_i= tf.nn.dropout(self.h_pool_flat_i, 1.0)
```
in model/deepconn.py

Thanks,

**Comment by sh0416** [comment]
It has different api for the tensorflow version, close issue..
